selfdrive/car/hyundai/values.py

```diff
@@ -845,25 +845,62 @@ class Buttons:
     ],
   },
   CAR.SANTA_FE_HEV_2022: {
+    (Ecu.abs, 0x7b0, None): [
+      b'\x00\x00\x00\x00',
+    ],
     (Ecu.fwdRadar, 0x7d0, None): [
       b'\xf1\x00TMhe SCC FHCUP      1.00 1.00 99110-CL500         ',
     ],
+    (Ecu.fwdRadar, 0x7e1, None): [
+      b'\xf1\x00TMhe SCC FHCUP      1.00 1.00 99110-CL500         ', 
+      b'"\xf1\x00',
+    ],
+    (Ecu.fwdRadar, 0x7e1, None): [
+      b'\xf1\x8799110CL500\xf1\x00TMhe SCC FHCUP      1.00 1.00 99110-CL500         ', 
+      b'"\xf1\x87\xf1\x81\xf1\x00',
```

Comment:
Not sure where any of these address/FW version pairs came from, going to revert and re-do for you